dao-go/assignments_test.go

```diff
@@ -2,14 +2,337 @@ package dao_test
+				assert.NilError(t, err)
+				nextPeriod, err := docgraph.LoadDocument(env.ctx, &env.api, env.DAO, nextPeriods[0].ToNode.String())
+				assert.NilError(t, err)
+				startTime, err = nextPeriod.GetContent("start_time")
+				assert.NilError(t, err)
+				firstPeriodEndSecs = int64(startTime.Impl.(eos.TimePoint))/1000000
+			}
+
+			//Create Adjustment 2.5 Periods after start period
+			CreateAdjustmentAfter(int64(50), 
+														firstPeriodStartSecs, 
+														env.PeriodDuration * 5 / 2,
+														&assignment,
+														&assignee, env, t)
+
+			//Create Adjustment 3.33 Periods after start period
+			CreateAdjustmentAfter(int64(100), 
+														firstPeriodStartSecs,
+														env.PeriodDuration * 10 / 3,
+														&assignment,
+														&assignee, env, t)
+
+			//Create Adjustment 3.66 Periods after start period
+			CreateAdjustmentAfter(int64(75), 
+														firstPeriodStartSecs,
+														env.PeriodDuration * 11 / 3,
+														&assignment,
+														&assignee, env, t)
+
+			//TODO: Calculate seeds_per_usd using tlosto.seeds table
+			hardcodedSeedsPerUsd := float32(39.0840)
+			var seedsDeferralFactor float32
+			{
+				settings, err := docgraph.GetLastDocumentOfEdge(env.ctx, &env.api, env.DAO, eos.Name("settings"))
+				
+				seedsDeferral, err := settings.GetContent("seeds_deferral_factor_x100")
+				assert.NilError(t, err)
+
+				seedsDeferralFactor = float32(seedsDeferral.Impl.(int64)) / 100.0
+			}
+
+			usdSalaryPerPhase := float32(3039.01)
+
+			//Number of HYPHA tokens received in 1 full period
+			totalHYPHA := float32(759.75)
+
+			//Number of HUSD tokens received in 1 full period
+			totalHUSD := float32(0)
+
+			//Number of HVOICE tokens received in 1 full period
+			totalHVOICE := float32(usdSalaryPerPhase*2.0)
+
+			//Number of seeds received in 1 full period
+			totalSEEDS := usdSalaryPerPhase * hardcodedSeedsPerUsd * seedsDeferralFactor
+
+			//Claim first period
+			t.Log("Waiting for a period to lapse...")
+			pause(t, env.PeriodPause, "", "Waiting...")
+
+			//This should get partial payment since the approved time should be > than 
+			//the start time of the period
+			_, err = ClaimNextPeriod(t, env, assignee.Member, assignment)
+			assert.NilError(t, err)
+
+			{
+				initTimeShare, err := docgraph.GetLastDocumentOfEdge(env.ctx, &env.api, env.DAO, eos.Name("initimeshare"))
+				assert.NilError(t, err)
+				approvedTime, err := initTimeShare.GetContent("start_date")
+				approvedSecs := int64(approvedTime.Impl.(eos.TimePoint))/1000000
+				periodDuration := float32(firstPeriodEndSecs-firstPeriodStartSecs)
+				timeFactor := float32(firstPeriodEndSecs-approvedSecs) / periodDuration
+				ValidateLastReceipt(int64(totalHUSD*timeFactor), 
+														int64(totalHYPHA*timeFactor), 
+														int64(totalHVOICE*timeFactor), 
+														int64(totalSEEDS*timeFactor), env, t)
```

Comment:
Could you pull this to the left? Same as above, is harder to read this way.

Reply:
Weird, I think this got somehow messed up by Visual Code auto formatting. 😆